### PR TITLE
Add blog post for FerretDB release v1.18.0

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md
Comment (AlekSi) [inline website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md:1]: https://github.com/FerretDB/FerretDB/actions/runs/7450730655/job/20270289936?pr=3950

File: website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+slug: new-ferretdb-v-11180-support-oplog-functionality
+title: FerretDB releases v1.18.0!
+authors: [alex]
+description: >
+  We just released FerretDB v1.18.0, which now supports `OpLog` functionality, capped collections, tailable cursors, among other new features.
```
Comment (AlekSi) [inline website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md:6]:
```suggestion
  We just released FerretDB v1.18.0, which now supports OpLog functionality, capped collections, tailable cursors, among other new features.
```

"OpLog" abbreviation is not different from, say, "DB"

File: website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+We are starting the new year with a major feature addition!
+The latest version of FerretDB v1.18.0 has just been released with support for basic `OpLog` functionality, along with other exciting features.
+
+`OpLog` functionality has been one of our most requested features, and it would make it possible for developers to build real-time applications using the Meteor framework.
+We made this a priority in the last quarter and are super excited that it's now available.
+We can't wait to see what you build with FerretDB!
+
+In addition to `OpLog` functionality, we've also added support for capped collections and tailable cursors.
+Other new features in the release include support for `createUser`, `dropUser`, `dropAllUsersFromDatabase`, and `UsersInfo`.
```
Comment (AlekSi) [inline website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md:21]: Those features are not ready to be used

File: website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+For example, to create a capped collection `testcollection` with a maximum size of 512MB, run:
+
+```js
+db.createCollection('testcollection', { capped: true, size: 536870912 })
+```
+
+This release also includes **support for tailable cursors**; both `tailable` and `awaitData` parameters are included.
+
+To add a tailable cursor, run:
```
Comment (AlekSi) [inline website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md:38]: We don't "add" cursors

File: website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+```js
+db.collection.find(
+  { $query: {}, $orderby: { $natural: 1 } },
```
Comment (AlekSi) [inline website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md:42]: That's not a valid syntax. Where does it come from?

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md (hunk 4 above)
Comment (Fashander) [reply website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md]: That's not supposed to be there. Thanks for sighting this, i should have deleted it before.

**Review by AlekSi — APPROVED** [review]

**Pull request merged**